### PR TITLE
chore: update README_ZH.md

### PR DESCRIPTION
**Pull request by w169376**
Update README_ZH.md

### Describe the pull request

A clear and concise description of what the pull request is about, i.e. what problem should be fixed? 

Link to the issue: 

### Checklist

- [x] I agree to follow the [Code of Conduct](https://go.dev/conduct) by submitting this pull request.
- [x] I have read and acknowledge the [Contributing guide](https://github.com/gogs/gogs/blob/main/.github/CONTRIBUTING.md).
- [x] I have added test cases to cover the new code.

### CODE REVIEW

**Comment by w169376** [comment]
Update README_ZH.md

**Comment by w169376** [comment]
Update README_ZH.md

File: README_ZH.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Gogs 是一款极易搭建的自助 Git 服务。
 
 ## 项目愿景
 
-Gogs（`/gɑgz/`）项目旨在打造一个以最简便的方式搭建简单、稳定和可扩展的自助 Git 服务。使用 Go 语言开发使得 Gogs 能够通过独立的二进制分发，并且支持 Go 语言支持的 **所有平台**，包括 Linux、macOS、Windows 以及 ARM 平台。
+Gogs（`/gɑgz/`）项目旨在打造一个以最简便的方式搭建简单、稳定和可扩展的自助 Git 服务。使用 Go 语言开发使得 Gogs 能够通过独立的二进制分发，并且支持 Go 语言支持的 **所有平台**，包括 Linux、macOS、Windows 和基于ARM的操作系统。
```
Comment (unknwon) [inline README_ZH.md:7]:
```suggestion
Gogs（`/gɑgz/`）项目旨在打造一个以最简便的方式搭建简单、稳定和可扩展的自助 Git 服务。使用 Go 语言开发使得 Gogs 能够通过独立的二进制分发，并且支持 Go 语言支持的 **所有平台**，包括 Linux、macOS、Windows 和基于 ARM 的操作系统。
```

**Review by unknwon — APPROVED** [review]
感谢提交！

**Pull request merged**